source/service.go

```diff
@@ -39,7 +39,8 @@ import (
 )
 
 const (
-	defaultTargetsCapacity = 10
+	defaultTargetsCapacity        = 10
+	awsLoadBalancerTypeAnnotation = "service.beta.kubernetes.io/aws-load-balancer-type"
```

Comment:
Please move `awsLoadBalancerTypeAnnotation` to https://github.com/kubernetes-sigs/external-dns/blob/master/source/source.go#L62 there should be already some examples.